tests/phpunit/elementor/modules/dev-tools/test-module.php

```diff
@@ -17,13 +16,13 @@ public function setUp() {
 		$this->module = new Module();
 	}
 
-	public function test_localize_settings() {
-		$this->module->deprecation->deprecated_function(__FUNCTION__, '0.0.0', '', '0.0.0'  );
```

Comment:
Where is this test?

Reply:
it's now extending App, there is no localize settings.

Reply:
Assert that the PHP deprecation was exposed to the JS

Reply:
Handled.

Reply:
Where?

Reply:
https://github.com/elementor/elementor/blob/master/tests/phpunit/elementor/modules/dev-tools/test-deprecation.php#L224